packages/react/src/ActionMenu/ActionMenu.tsx

```diff
@@ -217,9 +217,8 @@ const Anchor = React.forwardRef<HTMLElement, ActionMenuAnchorProps>(({children:
 })
 
 /** this component is syntactical sugar 🍭 */
-export type ActionMenuButtonProps = Omit<ButtonProps, 'children'> & {
-  children: React.ReactNode
-}
+export type ActionMenuButtonProps = ButtonProps
```

Comment:
Fixes: `Buttons must have discernible text [button-name]`

[for components-buttongroup-features--dropdown-split ](https://primer.style/react/storybook/?path=/story/components-buttongroup-features--dropdown-split)